LPS-142380 Use ReflectionUtil to keep the exception instead of wrapping it in a RuntimeException; in this change's case, the UI won't be able to show the error message if the exception is rethrown as RuntimeException.

jgarcialr opened the pull request:
- LPS-142380 Add validation in ExpandoColumnLocalServiceImpl
- LPS-142380 Add test
- LPS-142380 Use proper method to add the attribute with default value to make sure validation is executed before creating the value
- LPS-142380 Show error message on UI
- LPS-142380 buildLang
- LPS-142380 SV
- LPS-142380 SF, inline
- LPS-142380 SF, remove unnecessary try-rethrow
- LPS-142380 Use ReflectionUtil to keep the exception instead of wrapping it in a RuntimeException; in this change's case, the UI won't be able to show the error message if the exception is rethrown as RuntimeException.
- LPS-142380 SV
- LPS-142380 rename exceptions

crodriguezy committed:
LPS-142380 buildLang

crodriguezy committed:
LPS-142380 SV

crodriguezy committed:
LPS-142380 Add validation in ExpandoColumnLocalServiceImpl

crodriguezy committed:
LPS-142380 Add test

dantewang committed:
LPS-142380 Use proper method to add the attribute with default value to make sure validation is executed before creating the value

dantewang committed:
LPS-142380 Show error message on UI

dantewang committed:
LPS-142380 SF, inline

dantewang committed:
LPS-142380 SF, remove unnecessary try-rethrow

dantewang committed:
LPS-142380 Use ReflectionUtil to keep the exception instead of wrapping it in a RuntimeException; in this change's case, the UI won't be able to show the error message if the exception is rethrown as RuntimeException.

jgarcialr committed:
LPS-142380 SV

jgarcialr committed:
LPS-142380 rename exceptions

liferay-continuous-integration commented:
Closing pull request because all `liferay-portal` pullrequests sent to Brian Chan must be sent by using `ci:forward` on a pull request that was sent to someone else.

jgarcialr commented:
ci:reopen

jgarcialr commented:
Hi @brianchandotcom,

I've deleted the public constructor of ValueDataException.
I've replaced ValueDataException uses in other classes and grouped as new inner exceptions. Hope it's ok.

Regards

jgarcialr committed:
LPS-142380 Add validation in ExpandoValueLocalServiceImpl

liferay-continuous-integration commented:
Closing and locking pull request because pull requests sent to this user may not be updated. Please resend this pull request.
